common/src/main/java/io/netty/util/concurrent/DefaultThreadFactory.java

```diff
@@ -94,6 +95,11 @@ public DefaultThreadFactory(String poolName, boolean daemon, int priority) {
         prefix = poolName + '-' + poolId.incrementAndGet() + '-';
         this.daemon = daemon;
         this.priority = priority;
+        this.threadGroup = threadGroup;
```

Comment:
@zhangkun83 sorry for be a PITA but I just noticed we are missing a null check for threadGroup here.

Please do:
`this.threadGroup = ObjectUtil.checkNonNull(threadGroup);`